src/main/java/mate/academy/springboot/datajpa/service/CategoryService.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.springboot.datajpa.service;
+
+import mate.academy.springboot.datajpa.model.Category;
+
+public interface CategoryService {
+    Category add(Category category);
+
+    Category getById(Long id);
+
+    void deleteById(Long id);
+
+    Category put(Long id, Category category);
```

Comment:
You have `update` in one service, `put` in another for the same action. Use one name